draft-ietf-quic-datagram.md

```diff
@@ -247,6 +247,10 @@ ACK frames (within the limits specified by max_ack_delay) in reponse to receivin
 packets that only contain DATAGRAM frames, since the timing of these
 acknowledgements is not used for loss recovery.
 
+As with any ack-eliciting frame, when a sender suspects that a packet containing
+only DATAGRAM frames has been lost, it MAY send probe packets to elicit a faster
+acknowledgement as described in Section 6.2.4 of {{!RFC9002}}.
```

Comment:
6.2.4 says you MUST send a probe packet, so it sounds like you're describing a slight departure from RFC9002?  I have no objection to that, but this text doesn't read that way to me.

Possibly I'm misunderstanding what you're trying to say here?  Are you specifically trying to address the case where there are DATAGRAM frames in flight, but no data to send immediately?